deps/chakrashim/lib/chakra_shim.js

```diff
@@ -183,13 +183,15 @@
   //  err, func -- args from Error.captureStackTrace
   //  e -- a new Error object which already captured stack
   //  skipDepth -- known number of top frames to be skipped
-  function privateCaptureStackTrace(err, func, e, skipDepth) {
+  function privateCaptureStackTrace(err, func, e, skipDepth,
+    isLibraryCodeCaller) {
```

Comment:
Instead of adding and passing true for `isLibraryCodeCaller`, it looks `func` is not needed and you may pass `undefined` as func to achieve this?